assets/scss/books.scss

```
@@ -1,93 +1,155 @@
 
-  &.featured {
-    border-color: #478079;
-    box-shadow: 0 0 10px rgba(71, 128, 121, 0.2);
+// Note: the theme sets `body { overflow: hidden }` and scrolls via `html`
+// instead, which breaks native `position: sticky` (its containing block
+// resolves to the non-scrolling body). The nav is pinned via JS instead
+// (see static/js/books.js), toggling `.is-pinned` to switch to `fixed`.
+.book-nav {
+  --book-nav-bg: rgba(250, 250, 249, 0.72);
+  --book-nav-fade: rgba(250, 250, 249, 0.9);
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)